source/data-formats/dates-and-times.txt

```diff
@@ -0,0 +1,400 @@
+.. _pymongo-dates-times: 
+
+Dates and Times
+===============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: convert, span, central, mountain, pacific, eastern, calendar 
+
+Overview
+--------
+
+In this guide, you can learn how to handle Python ``datetime`` objects
+correctly in {+driver-short+}.
+
+.. note:: 
+   
+   With {+driver-short+}, you can't save ``datetime.date`` instances, because
+   there is no BSON type for dates without times. 
+   Convert all ``date`` objects to ``datetime`` objects before saving them to
+   MongoDB.
+
+Terminology
+-----------
+
+Python uses a dedicated data type, ``datetime.datetime``, to represent dates and times.
+MongoDB stores ``datetime`` values in *coordinated universal time (UTC)*,
+a global time standard local to London, England.
+
+A ``datetime`` value is *naive* when there is no supplemental information
+about its UTC offset or time zone. The following is an example of a naive ``datetime``
+object:
+
+.. code-block:: python
+
+   datetime(2002, 10, 27, 14, 0, 0)
+
+A ``datetime`` value is *aware* when it includes a ``tzinfo`` attribute that indicates
+the value's offset from UTC time, its time zone, and whether daylight saving time was in
+effect. The following is an example of an aware ``datetime`` object:
+
+.. code-block:: python
+
+   datetime(2002, 10, 27, 6, 0,
+            tzinfo=<DstTzInfo 'US/Pacific' PST-1 day, 16:00:00 STD>)
+
+*Localization* is the process of adding hours to, or subtracting hours from, a
+``datetime`` value to find its value in a specific time zone.
+
+To localize a ``datetime``, create a ``pytz.timezone`` object, passing in
```

Comment:
Note for tech reviewer: The [pytz docs](https://pypi.org/project/pytz/) suggest using the support now built into the standard library. Should we do that here? If so, what does it look like?